src/averageAges.js

```diff
@@ -37,7 +41,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(woman => woman.sex === 'f');
+
```

Comment:
Consider using a single filter for women with or without children, instead of filtering twice.